appIndicator.js

```diff
@@ -759,6 +776,7 @@ class AppIndicatorsIconActor extends St.Icon {
             ? SNIconType.ATTENTION : SNIconType.NORMAL;
 
         this._updateIconSize();
+        this._updateCustomIcons();
```

Comment:
Not sure it's worth to do this each time we reload an icon... I'd instead ensure that:
 - On `changed::custom-icons` we use `_updateCustomIcons` as callback and that we call invalidateicons afterwards
 - At startup we updateCustomIcons

Reply:
Ok I changed that. Now only at startup and if the settings are changed the updateCustomIcons function is called

Reply:
Actually I realized that my " fix" is not a fix. I'm not sure how to implement that it is run once at startup. As far as I understand it should be done somewhere in indicatorStatusIcon.js in the AppIndicatorsIndicatorTrayIcon class, but I don't see how I could call a method from the AppIndicatorsIconActor class from there